lib/puppet_x/templates/invoke_ps_command.erb

```diff
@@ -173,14 +369,25 @@ try
   $ps.Invoke()
 
   [Void]$ps.AddScript($powershell_code)
+  $ps.Commands.Commands[0].MergeMyResults(
```

Comment:
This, combined with the PSHostUserInterface and PSHostRawUserInterface, will output Info, Debug, Verbose, Warning to 'stdout'. The risk here is that we don't get PowerShell objects back, we get a string with output from all Streams merged together. Since we are looking for this, this is likely not a problem.